sympy/core/tests/test_args.py

```diff
@@ -4919,6 +4919,14 @@ def test_sympy__physics__optics__medium__Medium():
     from sympy.physics.optics import Medium
     assert _test_args(Medium('m'))
 
+@SKIP('should not be instantiated directly (only indirectly through instantiating Medium')
+def test_sympy__physics__optics__medium__MediumN():
+    pass
```

Comment:
You can still add a test that instantiates a MediumN indirectly.

Reply:
The only issue with directly instantiating `MediumN` is that it does not sympify its arguments (since `Medium`'s `__new__` sympifies arguments before passing them when creating the `MediumN` object. So, the arguments will have to be sympified first.

Reply:
I said *indirectly*